tests/test_md.py

```diff
@@ -391,6 +377,76 @@ def test_multiple_steps_local_validation(self):
         with pytest.raises(RuntimeError, match="k must be less than than 1000000.0"):
             ctxt.multiple_steps_local(100, np.array([1], dtype=np.int32), k=1e7)
 
+    def test_multiple_steps_local_selection_validation(self):
+        seed = 2022
+        np.random.seed(seed)
+
+        N = 8
+        D = 3
+
+        coords = np.random.rand(N, D).astype(dtype=np.float64) * 2
+        box = np.eye(3) * 3.0
+        masses = np.random.rand(N)
+
+        E = 2
+
+        params, potential = prepare_nb_system(
+            coords,
+            E,
+            p_scale=3.0,
+            cutoff=1.0,
+        )
+        nb_pot = potential.to_gpu(np.float32)
+
+        dt = 1.5e-3
+        radius = 1.2
+
+        v0 = np.zeros_like(coords)
+        bps = [nb_pot.bind(params).bound_impl]
+
+        # Use verlet since multiple_steps_local_selection doesn't depend on temperature
+        verlet = VelocityVerletIntegrator(dt, masses)
+        intg_impl = verlet.impl()
```

Comment:
```suggestion
        # Compatible with local NVE since multiple_steps_local_selection doesn't depend on temperature
        verlet = VelocityVerletIntegrator(dt, masses)
        intg_impl = verlet.impl()
```

Nice -- this interface also resolves an issue that was bugging me in https://github.com/proteneer/timemachine/pull/915#discussion_r1039794884